### ISSUE TITLE
(Carbon) zoom in photo mode doesn't remap

### ISSUE THREAD

**Issue by TorutheRedFox** [xan1242/NFS-XtendedInput#22]
the UI reacts to the change (the prompts change and respond to the presses and all), but the actual zooming action remains mapped to the triggers

**Comment by xan1242** [comment]
Yeah I know that actually, it's this issue: https://github.com/xan1242/NFS-XtendedInput/issues/12

This one is a duplicate of that technically speaking, so I'll mark it as such.

**Comment by xan1242** [comment]
One note for you @TorutheRedFox, if you're trying to map them like on the PS3, XInput doesn't support pressure sensitive buttons, while the PS3 version has L1 and R1 for zooming which also function with pressure sensitivity.

**Comment by TorutheRedFox** [comment]
yeah I know
